README.md

```diff
@@ -51,6 +51,7 @@ import ChipInput from 'material-ui-chip-input'
 | onRequestDelete | `function` | | Callback function that is called when a new chip was removed (in controlled mode). |
 | onTouchTap | `function` | | Callback function that is called when text input is clicked. |
 | onUpdateInput | `function` | | Callback function that is called when the input changes (useful for auto complete). |
+| onBlur | `function` | | Callback function that is called with `event` when the input loses focus, where `event.target.value` is the current value of the text input. |
```

Comment:
Please keep the alphabetic order of the props (makes the table easier to read). :wink: